### ISSUE TITLE
Documentation/utility required to be able to clear or modify a score

### ISSUE THREAD

**Issue by smfreegard** [haraka/haraka-plugin-karma#22]
There's no documentation on how to clear a karma score for an IP address in the case of false-positives or false-negatives.

**Comment by msimerson** [comment]
That's correct. From time to time I use FLUSHALL as a workaround.

**Comment by smfreegard** [comment]
Being able to clear individual history for IPs or ASNs would be far more useful than throwing everything away though.  Likewise being able to manually set a massive score for an IP or ASN might also be useful as a way of blacklisting in the case of false-negatives etc.

**Comment by msimerson** [comment]
I completely agree. It's just that the number of times I've needed to do it has been too few for me to prioritize an implementation. The only implementation I'm interested in doing is web based, and there's other scaffolding needed before I can do that.

**Comment by smfreegard** [comment]
Ok - I can think of a couple of times this very issue has come up on IRC though (in the case of false-positives).

**Comment by rricker1** [comment]
"From time to time I use FLUSHALL as a workaround."

I am trying to clear a false positive.  Where/how would I issue the FLUSHALL command?

Thank you

**Comment by msimerson** [comment]
In Redis db id 3.

**Comment by rricker1** [comment]
Thank you
